lib/dotcom/trip_plan/itinerary_row.ex

```diff
@@ -12,12 +12,14 @@ defmodule Dotcom.TripPlan.ItineraryRow do
   defmodule Dependencies do
     @moduledoc false
 
+    @stops_repo Application.compile_env!(:dotcom, :repo_modules)[:stops]
+
     @type stop_mapper :: (Stops.Stop.id_t() -> Stops.Stop.t() | nil)
     @type route_mapper :: (Routes.Route.id_t() -> Routes.Route.t() | nil)
     @type trip_mapper :: (Schedules.Trip.id_t() -> Schedules.Trip.t() | nil)
     @type alerts_repo :: (DateTime.t() -> [Alerts.Alert.t()] | nil)
 
-    defstruct stop_mapper: &Stops.Repo.get_parent/1,
+    defstruct stop_mapper: Function.capture(@stops_repo, :get_parent, 1),
```

Comment:
We could remove this mapper as well and just call `@stops_repo.get_parent` in the the code